src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -526,23 +578,23 @@ private void ReadEndOfCentralDirectory()
                 // If the EOCD has the minimum possible size (no zip file comment), then exactly the previous 4 bytes will contain the signature
                 // But if the EOCD has max possible size, the signature should be found somewhere in the previous 64K + 4 bytes
                 if (!ZipHelper.SeekBackwardsToSignature(_archiveStream,
-                        ZipEndOfCentralDirectoryBlock.SignatureConstant,
-                        ZipEndOfCentralDirectoryBlock.ZipFileCommentMaxLength + ZipEndOfCentralDirectoryBlock.SignatureSize))
```

Comment:
Seems that after your changes, `ZipEndOfCentralDirectoryBlock.SignatureSize` is no longer used anywhere. Can you please remove it?

Reply:
It's gone, thanks!